src/modules/nameSchema/docs/de_DE.md

```diff
@@ -0,0 +1,65 @@
+# Namens-Schemata
+
+## Variablen
+
+| Variable              | Beschreibung                                                                         | Verwendbar in Templates für |
+|-----------------------|--------------------------------------------------------------------------------------|-----------------------------|
+| `{{unitId}}`          | ID der Einheit, z.B. `78020362`                                                      | Einheiten                   |
+| `{{unitType}}`        | Typ der Einheit (z.B. `LF 20/01`).                                                   | Einheiten                   |
+| `{{unitTypeCustom}}`  | Benutzerdefinierte Fahrzeugklasse.                                                   | Einheiten                   |
+| `{{unitAlias}}`       | Alias-Bezeichnung der Einheit (vgl. Aliase), ansonsten identisch zu `unitType`.      | Einheiten                   |
+| `{{unitIndex}}`       | Aufsteigende Nummerierung des Fahrzeugs. Parametrisierung möglich.                   | Einheiten                   |
+| `{{buildingId}}`      | ID des Gebäudes, z.B. `11221784`                                                     | Einheiten, Gebäude          |
+| `{{buildingType}}`    | Typ  der Einheit (z.B. `Feuerwehrschule`)                                            | Einheiten, Gebäude          |
+| `{{buildingAlias}}`   | Alias-Bezeichnung des Gebäudes (vgl. Aliase), ansonsten identisch zu `buildingType`. | Einheiten, Gebäude          |
+| `{{buildingIndex}}`   | Aufsteigende Nummerierung des Gebäudes. Parametrisierung möglich.                    | Einheiten, Gebäude          |
+| `{{buildingCaption}}` | Name des Gebäudes                                                                    | Einheiten                   |
+
+### Parametrisierung
```

Comment:
Thats way to technical for the default user. More examples or simpler language. Not everbody knows what "variablen" and "parameters" mean.

Reply:
@Cr4zyc4k3 I'm planning to add a live-preview for the template as @jxn-30 suggested. This component could be extended in the future to include block etc.

Additionally a preset feature is planned, so that regular use-cases could be simply selected without fiddling with the templates.